{{cookiecutter.project_name}}/monitoring/SetUpDataMonitoringForInferenceTable.py

```diff
@@ -0,0 +1,131 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC
+# MAGIC # Data Monitoring for Inference Table
+# MAGIC see monitoring/README.md for details
+
+# COMMAND ----------
+
+# Install the Data Monitoring client library
+# TODO: Fill in wheel URL
+%pip install "{wheel_URL}"
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ### Required Fields
+# MAGIC TODO: Fill in and review required fields
+
+# COMMAND ----------
+
+# The name of the column that represents the inference timestamp (i.e. time of prediction) for this row. This column should be of TimestampType.
+timestamp_col = "ts"
+
+# The name of the column that represents the model version used to predict on this row. 
+# This should correspond to the MLflow registered model version (or any identifier unique to a specific trained model).
+model_version_col = "model_version"
+
+# The name of the column that holds the prediction (from inference) for this row.
+prediction_col = "prediction"
+
+# The type of ML problem that the model is solving. This should be either “classification” or “regression”.
+# Other problem types are not supported at this time. 
+problem_type = "regression"
+
+# The inference table holding data to be monitored. Table name format {catalog}.{schema}.{table}
+inference_table_name = "{catalog}.{schema}.{table}"
+
+# Name of the schema in which to create output tables. format {catalog}.{schema}
+output_schema_name = "{catalog}.{schema}"
+
+# List of granularities to use when aggregating data
+# into time windows based on their timestamp. Currently the following static granularities are supported:
+# {"5 minutes", "30 minutes", "1 hour", "1 day", "n week(s)", "1 month", "1 year"}.
+granularities = ["1 hour"]
+
+# List of Databricks entity names that are associated
+# with this table. Only following entities are supported:
+#    ["models:/registry_model_name"] links a model registry model to the monitored table.
+linked_entities = ["models:/prod-{{cookiecutter.model_name}}"]
```

Comment:
Related to above, you're already automatically filling this in via template so should we remove from required section?